src/containers/user-profile/profile-info/ProfileInfo.tsx

```diff
@@ -132,6 +137,31 @@ const ProfileInfo = ({ userData, myRole }: ProfileInfoProps) => {
       }
   ].filter(Boolean)
 
+  const {
+    response: listOfChats,
+    loading: isChatsLoading,
+    fetchData
+  } = useAxios<ChatResponse[]>({
+    service: () => chatService.getChats(),
```

Comment:
You can pass reference for this service function directly without additional function
```suggestion
    service: chatService.getChats,
```

Reply:
@ShadowOfTheSpace, it will not work because of typescript. Everywhere in the project it is written like this (either through a separate function like `getChats ` which was removed with the last comments, or through an arrow function).

So `()=>` is required here.